.travis.yml

```
@@ -3,23 +3,45 @@ sudo: required
 
 matrix:
   include:
-    - name: CentOS 6 x86_64 build
-      env: ARCH=x86_64 BUILD_TYPE=centos6
-      install: true
+    - name: x86_64 build
```

This rename is pretty pointless. We _are_ building on CentOS 6. This should be reflected here.

```suggestion
    - name: CentOS 6 x86_64 build
```